<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Samples count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="samplesCount">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btGenerateSamples">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libshvvisu/images/images.qrc">
           <normaloff>:/shv/visu/images/media-play.svg</normaloff>:/shv/visu/images/media-play.svg</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="twGraph">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="TimelineGraphWidget" name="tabTimeline">
       <attribute name="title">
        <string>Timeline</string>
       </attribute>
      </widget>
      <widget class="HistogramGraphWidget" name="tabHistogram">
       <attribute name="title">
        <string>Histogram</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimelineGraphWidget</class>
   <extends>QWidget</extends>
   <header>timelinegraphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistogramGraphWidget</class>
   <extends>QWidget</extends>
   <header>histogramgraphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../libshvvisu/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
